src/store/soundsStore.js

```diff
@@ -55,6 +60,76 @@ const mutations = {
 	setUserId(state, userId) {
 		state.userId = userId
 	},
+
+	/**
+	 * Sets the join audio object
+	 * 
```

Comment:
```suggestion
	 *
```

> ### GitHub Actions / eslint
>
> Trailing spaces not allowed                                              


Same for the other docs.